src/components/FormView.jsx

```diff
@@ -69,6 +70,12 @@ const FormView = ({
       fields_to_send.push(key);
     }
   }
+  if (data.captcha?.indexOf('norobots') >= 0) {
```

Comment:
Why not `if (data.captcha === 'norobots-captcha') {` ?
(IMO the same would be also for recaptcha/hcaptcha/...) \cc @giuliaghisini

Reply:
I just copied-over from recaptcha/hcaptcha but I think it would be better to look for exact matches.

Reply:
I modified this to look for an exact match.